src/Nri/Ui/QuestionBox/V4.elm

```diff
@@ -188,7 +188,7 @@ viewStandalone : Config msg -> Html msg
 viewStandalone config =
     div
         [ AttributesExtra.maybe Attributes.id config.id
-        , css config.containerCss
+        , css (config.containerCss ++ [ Css.fontSize (Css.px 18) ])
```

Comment:
Is this style actually necessary? Isn't the `viewBalloon` doing the styling?

Reply:
True, nice catch